$NetBSD: patch-dc,v 1.1 2007/05/28 08:13:25 dan Exp $
#
# old_revision [f92dd754bf5c1e6eddc9c462b8d68691cfeb7f8b]
#
# patch "diff_patch.hh"
#  from [f80e158dc7f4b50ac1ce4157896c226e95acfc54]
#    to [d481b1cf53caed0381272a18713c357880d3bd03]
#
============================================================
--- diff_patch.hh	f80e158dc7f4b50ac1ce4157896c226e95acfc54
+++ diff_patch.hh	d481b1cf53caed0381272a18713c357880d3bd03
@@ -48,6 +48,7 @@ content_merge_adaptor
                             file_id const & right_ident,
                             file_id const & merged_ident,
                             file_data const & left_data,
+                            file_data const & right_data,
                             file_data const & merged_data) = 0;
 
   virtual void get_ancestral_roster(node_id nid,
@@ -75,6 +76,7 @@ content_merge_database_adaptor
                     file_id const & right_ident,
                     file_id const & merged_ident,
                     file_data const & left_data,
+                    file_data const & right_data,
                     file_data const & merged_data);
 
   void get_ancestral_roster(node_id nid,
@@ -101,6 +103,7 @@ content_merge_workspace_adaptor
                     file_id const & right_ident,
                     file_id const & merged_ident,
                     file_data const & left_data,
+                    file_data const & right_data,
                     file_data const & merged_data);
 
   void get_ancestral_roster(node_id nid,
